tests/MSBuildDeviceIntegration/Resources/LinkDescTest/MainActivityReplacement.cs

```diff
@@ -111,7 +111,9 @@ protected override void OnCreate(Bundle bundle)
 			Android.Util.Log.Info(TAG, cldt.TryAccessNonXmlPreservedMethodOfLinkerModeFullClass());
 			Android.Util.Log.Info(TAG, LinkTestLib.Bug21578.MulticastOption_ShouldNotBeStripped());
 			Android.Util.Log.Info(TAG, LinkTestLib.Bug21578.MulticastOption_ShouldNotBeStripped2());
+#if !NET5_0
```

Comment:
Do we know if the `NET5_0` defines will continue to work in .NET 6? I forget what was ultimately decided there on the .NET SDK side...  I have been using `#if NET` in other places to try to handle future version upgrades, but I am not sure if this is the right approach.

Reply:
I just built a console app with a NET 6.0 alpha build and see:
```
/define:TRACE;DEBUG;NET;NET6_0;NETCOREAPP
```
Maybe we should use `NET or `NETCOREAPP` here instead.